RoslynPluginGenerator/RoslynPluginJarBuilder.cs

```diff
@@ -0,0 +1,394 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RoslynPluginJarBuilder.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+using SonarQube.Plugins.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace SonarQube.Plugins.Roslyn
+{
+    /// <summary>
+    /// Creates a SonarQube rules plugin
+    /// </summary>
+    public class RoslynPluginJarBuilder
```

Comment:
The structure and public API of this class is very similar to the RulesPluginBuilder class it replaces.
The main difference is in how it builds the jar.